docs/upgrade.html

```diff
@@ -17,6 +17,47 @@
 
 <h3><a id="upgrade" href="#upgrade">1.5 Upgrading From Previous Versions</a></h3>
 
+<h4><a id="upgrade_10" href="#upgrade_10">Upgrading from 0.9.x to 0.10.0.0</a></h4>
+0.10.0.0 has <a href="#upgrade_10_performance_impact">potential performance impact</a> and <a href="#upgrade_10_protocol_changes">protocol changes</a> (please review before upgrading).
```

Comment:
We want to make it clear that the performance impact is only during the upgrade period. Once the clients are upgraded or if people are just starting to use 0.10, there is no performance impact.